app/controllers/application_controller.rb

```diff
@@ -231,7 +231,7 @@ def show_offseason_banner?
   end
 
   def open_for_intake?
```

Comment:
I would prefer to remove this method, and inline it into the call sites. I find it confusing to have _both_ the config option _and_ the method, and when reviewing this, I've had to jump back and forth a lot to try to remember if this method has any special logic, but it doesn't.

Not a blocker.

Reply:
That's fine -- I'm trying to make the fewest changes necessary based on the original implementation because there's a tight timeline to get this done.